src/qt/pivx/send.cpp

```diff
@@ -761,13 +775,15 @@ void SendWidget::onDeleteClicked(){
                 it.remove();
             } else if (focusedEntry && entry->getNumber() > entryNumber){
                 entry->setNumber(entry->getNumber() - 1);
+                entry->setFocus();
             }
         }
 
         if (entries.size() == 1) {
             SendMultiRow* sendMultiRow = QMutableListIterator<SendMultiRow*>(entries).next();
             sendMultiRow->setNumber(entries.length());
             sendMultiRow->showLabels();
+            sendMultiRow->setFocus();
```

Comment:
Same as above, better to unify this and use the `setFocusOnLastRecipient` that you created.
Plus, move it to the bottom of the method. The last entry will get focus anyway (only one call inside this method).